config/exlager.schema.exs

```diff
@@ -4,165 +4,193 @@
       doc: """
       Choose the logging level for the journal backend.
       """,
-      to: "lager.handlers",
+      to: "lager.handlers.journal.level",
       datatype: [enum: [:emerg, :alert, :crit, :error, :warning, :notive, :info, :debug, :false]],
       default: :false
     ],
     "log.console.level": [
       doc: """
       Choose the logging level for the console backend.
       """,
-      to: "lager.handlers",
+      to: "lager.handlers.console.level",
       datatype: [enum: [:info, :error, :false]],
       default: :info
     ],
     "log.gelf.level": [
       doc: """
       Choose the logging level for the graylog backend.
       """,
-      to: "lager.handlers",
+      to: "lager.handlers.gelf.level",
       datatype: [enum: [:emerg, :alert, :crit, :error, :warning, :notive, :info, :debug, :false]],
       default: :false
     ],
     "log.gelf.host": [
       doc: """
       Hostname of the graylog server to forward log messages to.
       """,
-      to: "lager.handlers",
-      datatype: :string,
+      to: "lager.handlers.gelf.host",
+      datatype: :charlist,
```

Comment:
Isn't there `:string` type anymore? Should we convert `:charlist` on translation?